ksqldb-engine/src/main/java/io/confluent/ksql/query/PullQueryWriteStream.java

```diff
@@ -97,12 +99,6 @@ public PullQueryWriteStream(
   ) {
     this.queryLimit = queryLimit;
     this.translator = translator;
-
-    // register a drainHandler that will wake up anyone waiting on hasCapacity
```

Comment:
I don't see how this is necessary, since every time we call the drain handlers, we anyways acquire the monitor.